support-e2e/tests/genericCheckout.test.ts

```diff
@@ -11,38 +11,32 @@ import { afterEachTasks } from './utils/afterEachTest';
 const testDetails = [
 	{
 		product: 'Contribution',
+		price: 9,
 		ratePlan: 'Monthly',
 		paymentType: 'Direct debit',
-		price: 9,
-		country: 'UK',
-		glyph: '£',
+		internationalisationId: 'uk',
 		paymentFrequency: 'month',
 	},
 	{
 		product: 'Contribution',
+		price: 90,
 		ratePlan: 'Annual',
 		paymentType: 'Credit/Debit card',
-		price: 90,
-		country: 'US',
-		glyph: '$',
+		internationalisationId: 'us',
 		paymentFrequency: 'year',
 	},
 	{
 		product: 'SupporterPlus',
 		ratePlan: 'Monthly',
 		paymentType: 'PayPal',
-		price: 17,
```

Comment:
I've removed `price` here as it's not relevant to SupporterPlus, we get that value from the catalog.

Reply:
This must be where  the tests failed would guess.

Reply:
Part of it.